gluoncv/model_zoo/center_net/deconv_dla.py

```diff
@@ -0,0 +1,288 @@
+                node = CustomConv(out_channels, out_channels, use_dcnv2=use_dcnv2,
+                                  norm_layer=norm_layer, norm_kwargs=norm_kwargs)
+
+                up = nn.Conv2DTranspose(in_channels=out_channels, channels=out_channels,
+                                        kernel_size=f * 2, strides=f,
+                                        padding=f // 2, output_padding=0,
+                                        groups=out_channels, use_bias=False,
+                                        weight_initializer=BilinearUpSample())
+
+                self.projs.add(proj)
+                self.ups.add(up)
+                self.nodes.add(node)
+
+
+    def hybrid_forward(self, F, layers):
+        for i in range(self.startp + 1, self.endp):
+            upsample = self.ups[i - self.startp - 1]
+            project = self.projs[i - self.startp - 1]
+            layers[i] = upsample(project(layers[i]))
+            node = self.nodes[i - self.startp - 1]
+            layers[i] = node(layers[i] + layers[i - 1])
+        return layers
+
+class DLAUp(nn.HybridBlock):
+    """Deep layer aggregation upsampling layer.
+
+    Parameters
+    ----------
+    startp : int
+        Start index.
+    channels : iterable of int
+        Output channels.
+    scales : iterable of int
+        Upsampling scales.
+    in_channels : iterable of int
+        Input channels.
+    use_dcnv2 : bool
+        Whether use ModulatedDeformableConvolution(DCN version 2).
+    norm_layer : nn.HybridBlock, default is BatchNorm
+        The norm layer type.
+    norm_kwargs : dict
+        Extra arguments to norm layer.
+
+    """
+    def __init__(self, startp, channels, scales, in_channels=None,
+                 use_dcnv2=False, norm_layer=nn.BatchNorm, norm_kwargs=None, **kwargs):
+        super(DLAUp, self).__init__(**kwargs)
+        self.startp = startp
+        if in_channels is None:
+            in_channels = channels
+        self.channels = channels
+        channels = list(channels)
+        scales = np.array(scales, dtype=int)
+        with self.name_scope():
+            self.idas = nn.HybridSequential('ida')
+            for i in range(len(channels) - 1):
+                j = -i - 2
+                self.idas.add(IDAUp(channels[j], in_channels[j:],
+                                    scales[j:] // scales[j], use_dcnv2=use_dcnv2,
+                                    norm_layer=norm_layer, norm_kwargs=norm_kwargs))
+                scales[j + 1:] = scales[j]
+                in_channels[j + 1:] = [channels[j] for _ in channels[j + 1:]]
+
+    def hybrid_forward(self, F, layers):
+        out = [layers[-1]] # start with 32
+        for i in range(len(layers) - self.startp - 1):
+            ida = self.idas[i]
+            ida.startp = len(layers) -i - 2
+            ida.endp = len(layers)
+            layers = ida(layers)
+            out.insert(0, layers[-1])
+        return out
+
+
+class DLASeg(nn.HybridBlock):
+    """Short summary.
```

Comment:
Can you provide more details for DLASeg?

Reply:
done